src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,11 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null
```

Comment:
handle it to upper component
```suggestion
  user: User
```